src/style.css

```
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 4px;
+  width: 96px;
+}
+
+.stars_star {
```

according to the BEM
```suggestion
.stars__star {
```